microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,410 @@ ms.author: msedgedevrel
+   ```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.  Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine, and can result in unexpected behavior.
+
+
+##### [Environment variable](#tab/environment-variable)
 
-   Replace `WebView2APISample.exe` with the file name of your own app executable or the Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
+Name: `WEBVIEW2_RELEASE_CHANNELS`  <!-- 2 trailing spaces = linebreak -->
+Value: `<string of desired channels (ex: 0,1)>`
 
-#### Resuming using the default, WebView2 Evergreen Runtime
+The target app can't be set when using the environment variable approach.  Therefore, when set as a global environment, this setting will affect all apps that use WebView2 on the machine.
 
-To delete the `ReleaseChannelPreference` registry override, run the command:
 
-`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /f`
+##### [Group policy](#tab/group-policy)
 
+Set the `ReleaseChannel` policy.
+
+* Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`  <!-- 2 trailing spaces = linebreak -->
+* Value: `<string of desired channels (ex: 0,1)>`
+
+Do either of the following:
+
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  For more information, see [Configure Microsoft Edge policy settings on Windows devices](/deployedge/configure-microsoft-edge).
+
+* Use the built-in policy on Intune.  For more information, see [Configure Microsoft Edge policy settings in Microsoft Intune](/mem/intune/configuration/administrative-templates-configure-edge).
+
+---
 
-<!-- 3. Environment Variable ============================================== -->
-## Using an environment variable
+<!-- ------------------------------ -->
+#### How to use an environment variable
 
-To make your application use a Microsoft Edge preview channel by using an environment variable:
 
-1. In the Windows search bar, enter "environment", and then select **Edit the system environment variables**.
+<!-- ---------- -->
+###### Setting environmental variables using API
+
+You can set environment variables by using `SetEnvironmentVariable`.
+
+* .NET: [Environment.SetEnvironmentVariable Method](/dotnet/api/system.environment.setenvironmentvariable)
+
+* WinRT: [EnvironmentManager.SetEnvironmentVariable Method](/windows/windows-app-sdk/api/winrt/microsoft.windows.system.environmentmanager.setenvironmentvariable)
+
+* Win32: [SetEnvironmentVariable function (winbase.h)](/windows/win32/api/winbase/nf-winbase-setenvironmentvariable)
+
+
+<!-- ---------- -->
+###### Setting environmental variables globally
```

Comment:
```suggestion
###### Setting environmental variables globally

You can also set environment variables by using the Windows system UI. If you do, it will make the scope of the environment variable global for the current user or system, and will affect all apps that create WebView2 controls.
```

Reply:
```
###### Setting environmental variables globally

You can set environment variables by using the Windows system UI.  
This approach makes the scope of the environment variable global for the current user or system, 
and affects all apps that create WebView2 controls.

1. In the Windows search bar, enter "environment", and then select **Edit the system environment variables**:
```